src/components/userProfile/EditProfile.js

```diff
@@ -25,7 +25,7 @@ export class EditProfile extends Component {
   componentWillReceiveProps(nextProps) {
     if (nextProps.upDateSuccess === true) {
       const { history } = nextProps;
-      history.push("/profile");
+      history.push(`/profile/${sessionStorage.getItem("username")}`);
```

Comment:
What is this?

Reply:
Hey Emma, since you worked on getting a single users profile, getting other users profiles highly relied on your files as your task was oblivious of how to get other users profiles so I had to change it in a way to facilitate that. The session was included in the profiles routes in order to be able to navigate to other users profiles and not just your own.